draft-iab-protocol-maintenance.md

```diff
@@ -162,7 +218,9 @@ This premise that specifications will be imperfect is correct.  However, the
 robustness principle is almost always the incorrect solution to the problem.
 
 
-# Protocol Decay {#decay}
+# Harmful Consequences of the Robustness Principle
+
+## Protocol Decay {#decay}
```

Comment:
So this violates an important rule.  Do not create an empty section.

Can we maybe reuse some of the text you dropped?

```suggestion
# Harmful Consequences of the Robustness Principle

Problems in other implementations can create an unavoidable need to temporarily
apply the robustness principle.  However, even temporary use carries risks.


## Protocol Decay {#decay}
```